src/utils/formattingUtils.test.ts

```diff
@@ -0,0 +1,25 @@
+import { formatNumber } from 'src/utils/formattingUtils';
```

Comment:
should we add few more tests that are testing that the position argument works as expected? :)